.changeset/strange-dots-draw.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lion/ui': patch
+---
+
+narrowed the scope of the selectors that query [slot=invoker] and [slot=content] to prevent that any nested elements with [slot=invoker] and [slot=content] are moved to slot=\_accordion as well
```

Comment:
Should start with the element e.g.

```suggestion
`accordion`: narrowed the scope of the selectors that query [slot=invoker] and [slot=content] to prevent that any nested elements with [slot=invoker] and [slot=content] are moved to slot=\_accordion as well
```